src/sdp/DefaultSDP.ts

```diff
@@ -267,6 +267,20 @@ export default class DefaultSDP implements SDP {
     return new DefaultSDP(originalSdp);
   }
 
+  withVideoLayersAllocationRtpHeaderExtension(): DefaultSDP {
```

Comment:
Where is this method called?

Reply:
It is called in SetLocalDescriptionTask to negotiate with the back end